deployment/aws-config/config.yaml.template

```diff
@@ -86,3 +86,6 @@ RDS_ROLE: "refinery"
 
 # The instance type for the EC2 instance
 # FIXED to "m3.medium"
+
+# Redirects all non-HTTPS requests to HTTPS, False is default.
+SECURE_SSL_REDIRECT = True
```

Comment:
Should remove this setting? Ref: https://github.com/refinery-platform/refinery-platform/pull/2586

Reply:
Yes. Branch switching mistake. Will remove.